build.gradle.kts

```diff
@@ -215,6 +231,8 @@ tasks {
 		useJUnitPlatform()
 		filter {
 			includeTestsMatching("*Tests")
+			// included for documentation purposes
+			// includeTestsMatching("*Demo")
```

Comment:
Is this correct?

```suggestion
			includeTestsMatching("*Demo")
```

Other than that, I think we can merge.

Reply:
Nope. This is just dead code.